theano/tensor/basic.py

```diff
@@ -3055,7 +3055,9 @@ def var(input, axis=None, keepdims=False):
     centered_input = input - mean_input
 
     # return the mean sqr
-    return mean((centered_input ** 2), axis, keepdims=keepdims)
+    v = mean((centered_input ** 2), axis, keepdims=keepdims)
+    v.name = 'var'
+    return v
```

Comment:
> I don't understand the correction needed in the docstring.

The docstring for `std()` literally says "It calls var() and var()." That's obviously not what it does. Maybe it was meant to say "It calls var() and sqrt()." I just noticed that while checking if its result could also get a name.